mne/decoding/tests/test_base.py

```diff
@@ -61,13 +61,32 @@ def test_get_coef():
     from sklearn.base import TransformerMixin, BaseEstimator
     from sklearn.pipeline import make_pipeline
     from sklearn.preprocessing import StandardScaler
+    from sklearn import svm
     from sklearn.linear_model import Ridge, LinearRegression
-
-    lm = LinearModel()
-    assert (is_classifier(lm))
-
-    lm = LinearModel(Ridge())
-    assert (is_regressor(lm))
+    from sklearn.model_selection import GridSearchCV
+
+    lm_classification = LinearModel()
+    assert (is_classifier(lm_classification))
+
+    lm_regression = LinearModel(Ridge())
+    assert (is_regressor(lm_regression))
+
+    parameters = {'kernel': ['linear'], 'C': [1, 10, 100, 1000]}
+    lm_gs_classification = LinearModel(
+        GridSearchCV(
+            svm.SVC(), parameters,
+            cv=5, refit=True, iid=False, n_jobs=5
```

Comment:
for testing try to use something as fast as possible. So cv=2. Also n_jobs should be set to 1 for testing.